taskcluster/ci/lint/kind.yml

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+job-defaults:
+    attributes:
+        code-review: true
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: base}
+        max-run-time: 7200
+    run:
+        use-caches: false
```

Comment:
Why don't you want to use caches for this task?

Reply:
That's a great question. I primarily copied what @tomprince did in reference-browser: https://github.com/mozilla-mobile/reference-browser/blob/4748d9e766d62c5156829cb1ed72c73ba996582c/taskcluster/ci/lint/kind.yml#L20

After thinking about it more thoroughly, we run `gradlew clean` before each of these tasks, so mounting the cache doesn't provide any value. Do I get that right, @tomprince ? 

Anyway, I'm happy to make the change in a followup.

Reply:
It has been awhile since I did this, but I suspect the reason is I didn't want to think too hard about scopes for docker-worker caches at the time, and then never revisited it.